src/app/features/swap-form/reducer.ts

```diff
@@ -128,20 +129,14 @@ export type SliceState = {
       };
       status: ThunkStatus;
     };
-  };
-  // State of cashflow calculator
-  cashflowCalculator: {
-    // User-inputted predicted variable apy
-    predictedApy: string;
-    // Cached variable factor from termStart to now
-    variableFactorStartNow: {
-      value: number;
+    cashflowInfo: {
+      averageFixedRate: number;
+      accruedCashflowExistingPosition: number;
+      accruedCashflowEditPosition: number;
+      estimatedAdditionalCashflow: (estimatedApy: number) => number;
```

Comment:
this is kinda bad practice in redux. Why do we keep functions in state? Is there no way around this? Perhaps we can create utilities that perform this? Try to avoid this at all cost!